src/app/donation-start/donation-start-container/donation-start-container.component.ts

```diff
@@ -31,23 +31,22 @@ export class DonationStartContainerComponent implements OnInit{
   ) {
   }
 
-   ngOnInit() {
+  ngOnInit() {
     this.campaign = this.route.snapshot.data.campaign;
-     this.campaignOpenOnLoad = this.campaignIsOpen();
-     this.imageService.getImageUri(this.campaign.bannerUri, 830).subscribe(uri => this.bannerUri = uri);
+    this.campaignOpenOnLoad = this.campaignIsOpen();
+    this.imageService.getImageUri(this.campaign.bannerUri, 830).subscribe(uri => this.bannerUri = uri);
+  }
 
-     const idAndJWT = this.identityService.getIdAndJWT();
-     if (idAndJWT) {
-       this.loadAuthedPersonInfo(idAndJWT.id, idAndJWT.jwt);
-     } else {
-       if (!this.donationStartForm) {
-         console.error("Donation start form not loaded");
-       }
-       // this.donationStartForm is undefined when we're running donation-start-login.component.spec.ts . I'm not sure
```

Comment:
This was a result of the DonationStartContainer component test having the wrong name in its description.